rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -777,13 +778,16 @@ void main_window::HandlePupInstallation(QString file_path)
 		return;
 	}
 
-	if (!m_gui_settings->GetBootConfirmation(this))
+	if (!Emu.IsStopped() && !Emu.IsReady())
```

Comment:
This is technically a regression, as people might also try to install the firmware while in ready state.

Reply:
Fixed, added create_cache optional parameter.